runhouse/servers/cluster_servlet.py

```diff
@@ -330,6 +336,25 @@ async def acheck_cluster_logs(self, interval_size: int):
 
         return logs_den_resp, new_start_log_line, new_end_log_line
 
+    async def aperiodic_autostop_check(self):
+        """Periodically check the autostop of the cluster"""
+        interval_size = 60
+        while True:
+            should_update_autostop: bool = self.autostop_helper is not None
+            if not should_update_autostop:
```

Comment:
what if we move this into the try as follows. this way, we still have the sleep for interval size apply to all cases.

```
try:
    if should_udpate_autostop:
          status, _ = await ...
          ...
except...
finally...
```